### PR TITLE
Ks 2021 05 disable unload warning outside dashboard

### LINKED ISSUE #1226: Browser says I haven´t sent information
**URL:** https://kosmo-dev.liqd.net/liqd/subjects/2021-00017/
**user:** admin
**expected behaviour:** I reported a comment and then just go via the "back" button back to the discussion
**behaviour:** My browser warns me that I haven´t sent the information that I put in - but I did!
**important screensize:** Desktop
**device & browser:** MacBook, Mozilla
**Comment/Question:** Is this something in our hands or is it something with the Browser I use?

Bildschirmfoto 2021-04-26 um 11 30 02

### PR DESCRIPTION
**Pull request by Rineee**
fixes #1226

Although this is not the proper way, I guess we should find out why the browser thinks the information in Report Modal is not sent, even though it is..

### CODE REVIEW

File: adhocracy-plus/templates/a4dashboard/base_dashboard_project.html [hunk 1]
```diff
@@ -138,3 +138,8 @@
         </div>
     </div>
 {% endblock %}
+
+{% block extra_js %}
```
Comment (fuzzylogic2000) [inline adhocracy-plus/templates/a4dashboard/base_dashboard_project.html:142]: Do we also have a dashboard_base? Or will we also get problems when we use that?

**Review by fuzzylogic2000 — COMMENTED** [review]
Looking good! Just one question.

File: adhocracy-plus/templates/a4dashboard/base_dashboard_project.html (hunk 1 above)
Comment (Rineee) [reply adhocracy-plus/templates/a4dashboard/base_dashboard_project.html:142]: Yes, we do have a dashboard_base. I just did it the way its done in meinBerlin, there its added to base_dashboard_project. But I actually dont think we get problems if I add it to dashboard base. So should I do that instead?

Comment (fuzzylogic2000) [reply adhocracy-plus/templates/a4dashboard/base_dashboard_project.html:142]: As you like! You can just merge or move it to dashboard_base. Well, I think, I prefer the latter! :)

**Review by fuzzylogic2000 — APPROVED** [review]
:unlock:

**Pull request merged**